packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -367,7 +367,7 @@ export class TextContentResource implements Resource {
             return content;
         } else {
             const content = await this.proxy.$provideTextDocumentContent(this.uri.toString());
-            if (content) {
+            if (typeof content === 'string') {
                 return content;
```

Comment:
Should we not simply return a default value if `content` is `undefined` or `null` like:

```suggestion
            return content ?? '';
```

Reply:
Yeah, that's probably the better solution for that issue.